doc/stdlib/linalg.rst

```diff
@@ -481,6 +481,24 @@ Linear algebra functions in Julia are largely implemented by calling functions f
 
    Optionally takes a ``thin`` Boolean argument, which if ``true`` omits the columns that span the rows of ``R`` in the QR factorization that are zero. The resulting matrix is the ``Q`` in a thin QR factorization (sometimes called the reduced QR factorization). If ``false``\ , returns a ``Q`` that spans all rows of ``R`` in its corresponding QR factorization.
 
+.. function:: lqfact!(A) -> LQ
+
+   .. Docstring generated from Julia source
+
+   Compute the LQ factorization of ``A``\ , using the input matrix as a workspace. See also :func:`lq\ .
```

Comment:
Missing closing backtick after `` :func:`lq ``? Should be `` [`lq`](:func:`lq`) `` in the source docstrings.